packages/amplify-graphql-index-transformer/src/graphql-index-transformer.ts

```diff
@@ -54,7 +56,16 @@ export class IndexTransformer extends TransformerPluginBase {
     this.directiveList.push(args);
   };
 
-  // TODO(cjihrig): before() and after() are needed to handle sync queries once they are supported.
+  public after = (ctx: TransformerContextProvider): void => {
+    if (!ctx.isProjectUsingDataStore() || !this.resolverMap.size) return;
```

Comment:
It might make the code a bit simpler to drop the `!this.resolverMap.size` check here. The `forEach()` loop below won't do any work and any performance benefit should be negligible.

Reply:
updated